my-profile-webapp/src/main/java/edu/wisc/my/profile/web/LocalContactInformationController.java

```diff
@@ -0,0 +1,58 @@
+package edu.wisc.my.profile.web;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import edu.wisc.my.profile.model.ContactInformation;
+import edu.wisc.my.profile.service.LocalContactInformationService;
+
+@Controller
+@RequestMapping("/localContactInfo")
+public class LocalContactInformationController {
+  
+  protected static final Logger logger = LoggerFactory.getLogger(LocalContactInformationController.class);
```

Comment:
It's a non-final class, so I'd have gone with a 

`protected final Logger logger = LoggerFactory.getLogger(getClass());`

but I recognize that reasonable people differ as to whether that's a best practice.